### ISSUE TITLE
Suport line-by-line

### ISSUE THREAD

**Issue by AndreaCrotti** [inglesp/prescons#3]
I think sometimes it might be useful to go forward line by line,
instead of block by block.

For example using ENTER we might go the whole function, and space line
by line, or vice-versa..

What do you think?
Thanks

**Comment by inglesp** [comment]
Hi Andrea,

Sounds reasonable, and I'd happily accept a patch for that.  I think that <enter> for a whole function, and <space> for a single line would be the right way round.

Peter.

PS thanks for giving this a mention on the TIP list!

**Comment by inglesp** [comment]
Also, I see your own fork is coming off 8ac0697, and that you've reimplemented Python 3 support -- this is stuff that's already in master.

**Comment by AndreaCrotti** [comment]
Yes sorry I forked another fork by mistake, now I re-forked the right one so I will contribute to this only.. thanks

**Comment by inglesp** [comment]
Any pull request for this?
